src/extension/debug-page/typeson.ts

```diff
@@ -0,0 +1,16 @@
+import Typeson from 'typeson'
+
+// @ts-ignore
+import builtins from 'typeson-registry/dist/presets/builtin' // @ts-ignore
+import num from 'typeson-registry/dist/presets/special-numbers' // @ts-ignore
+import blob from 'typeson-registry/dist/types/blob' // @ts-ignore
+import file from 'typeson-registry/dist/types/file' // @ts-ignore
+import fileList from 'typeson-registry/dist/types/filelist' // @ts-ignore
+import imageBitMap from 'typeson-registry/dist/types/imagebitmap' // @ts-ignore
+
+const typeson = new Typeson({})
+typeson.register(builtins)
+typeson.register(num)
+typeson.register([blob, file, fileList, imageBitMap, num])
```

Comment:
Please also write a serializer for `CryptoKey`. Notice, some `CryptoKey` might not be exported (by design).